packages/error-view/README.md

```diff
@@ -0,0 +1,7 @@
+# Error View
+
+A component used to display an error view. For further details on this component, see [docs.mdx](./docs.mdx). These docs can also be viewed through our [Swingset instance](https://react-components.vercel.app).
+
+## `useErrorPageAnalytics`
```

Comment:
Kept this README pretty minimal, to avoid duplicating content in the `docs.mdx` file.